README.md

```diff
@@ -23,7 +23,7 @@ If you have any question of us source code maintainers, please feel free to reac
 
 # Contributing
 
-Visit our [CONTRIBUTING.md](https://github.com/maebeale/mutual-aid-app/blob/master/CONTRIBUTING.md) file for more information on making your contribution. We look forward to it!
+Visit our [CONTRIBUTING.md](https://github.com/rubyforgood/mutual-aid-app/blob/master/CONTRIBUTING.md) file for more information on making your contribution. We look forward to it!
```

Comment:
NBD, but i learned recently that in-rep files can be [relative-linked](https://github.blog/2013-01-31-relative-links-in-markup-files/)